src/schema/v2/CollectorProfile/collectorProfile.ts

```diff
@@ -59,7 +59,7 @@ export const CollectorProfileFields: GraphQLFieldConfigMap<
 
   // moved InquirerCollectorProfileFields here
   location: { type: myLocationType },
-  artsyUserSince: date,
```

Comment:
Out of ignorance: will this be an issue with the already existing usages of this field in the FE?

Reply:
you can still query the field without the format filter, it's optional so won't break 😉 
I tested locally too to be extra safe 😂